nautobot_golden_config/views.py

```diff
@@ -188,7 +188,7 @@ def alter_queryset(self, request):
         return pivot(
             self.queryset,
             ["device", "device__name"],
-            "rule__feature__slug",
+            "rule__feature__name",
```

Comment:
I am not following why this change is advantageous?

Reply:
The way in which the rule__feature is called here and in the `ConfigComplianceTable` in table.py had to match, in order to determine a successful or not compliance. Since one was being called by a slug and the other by name, we decided to always use the name attribute. We ran this by Jeff K. Since the slug and name were being referenced in different files, there was a bug where the rendering of the table was not correctly displaying the Checkmark or Red X, depending on compliance result as the objects did not match.  This prompted for us to auto generate the slug in the form as well.

Reply:
ok, yes, for sure an issue if referencing slug in one place and name in the other. Can you point me to the name reference? My gut says that slug should be the correct data point to view.

Reply:
I believe it's here: [tables.py](https://github.com/nautobot/nautobot-plugin-golden-config/blob/102bafa1cbdb4442850c844d406055656a0e3813/nautobot_golden_config/tables.py#L136)

Reply:
I am inclined to think about slug, what say you @jeffkala ?

Reply:
I said name because it's using that on the column names on the table.  So when we changed it to slug during testing the column names to me were less "human readable" as it was the slug instead of the name the user defined.  I'm happy to be overruled.